cmd/autoheal/awx_job.go

```diff
@@ -47,7 +49,8 @@ func (h *Healer) runAWXJob(rule *monitoring.HealingRule, action *monitoring.AWXJ
 		Proxy(awxProxy).
 		Username(awxUser).
 		Password(awxPassword).
-		Insecure(true).
+		CACrt(awxCA).
```

Comment:
`Crt` is a bit cryptic for a public method name, in my opinion. Can we use something more readable? Maybe `CACertificates`?